testsuite/MDAnalysisTests/util.py

```diff
@@ -262,3 +262,12 @@ def get_userid():
         return 1000
     else:
         return os.geteuid()
+
+def get_running_dask_client():
+    # the solution found here: https://stackoverflow.com/questions/59070260/dask-client-detect-local-default-cluster-already-running
+    # note: the used API is non-public and can change any time
+    try:
+        import dask.distributed
+        return dask.distributed.client._get_global_client()
+    except ImportError:
+        return None
```

Comment:
Is this the behaviour we want? Since this function only makes sense with `dask.distributed`, maybe it would be better to propagate the exception (with a nice error message).

Reply:
I think the logic is slightly different now, but I'm not sure what you mean, to be honest